packages/hooks/src/__tests__/useGroupedFiatTransactions.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import { renderHook } from '@testing-library/react-hooks';
+import useGroupedFiatTransactions from '../useGroupedFiatTransactions';
+
+describe('useGroupedFiatTransactions', () => {
```

Comment:
@rostislav-deriv This test doesn't make sense 🤔
There is no use of store in the hook then why mocking store? The hook is using API, You would need to mock `useFetch` for it. 

Please double check this test file 🙏🏻

Reply:
No more mocking the store in this file